builds/jazz_azure-create-service/index.js

```diff
@@ -1,5 +1,4 @@
 
```

Comment:
Looks like this module uses deprecated azure modules. Let's check if we can use the new azure node sdk - https://github.com/azure/azure-sdk-for-js
```
############################################################################
# Active development of this SDK has been moved to [31m@azure/arm-cdn[0m[1m package. #
# This package is in maintenance mode and will be deprecated in July 2019. #
#  Visit https://aka.ms/azure-sdk-for-js-migration for more information.   #
############################################################################

[0m
[1m

#################################################################################
# Active development of this SDK has been moved to [31m@azure/arm-eventhub[0m[1m package. #
#   This package is in maintenance mode and will be deprecated in July 2019.    #
#     Visit https://aka.ms/azure-sdk-for-js-migration for more information.     #
#################################################################################

[0m
[1m

###################################################################################
# Active development of this SDK has been moved to [31m@azure/arm-servicebus[0m[1m package. #
#    This package is in maintenance mode and will be deprecated in July 2019.     #
#      Visit https://aka.ms/azure-sdk-for-js-migration for more information.      #
###################################################################################```

Reply:
Following are the modules that might need to be migrated - 

```
    "azure-arm-apimanagement": "^4.4.0-preview",
    "azure-arm-cdn": "^4.1.0",
    "azure-arm-cosmosdb": "^2.3.0",
    "azure-arm-eventhub": "^3.1.0",
    "azure-arm-resource": "^7.3.0",
    "azure-arm-sb": "^3.1.0",
    "azure-arm-storage": "^7.0.0",
    "azure-arm-website": "^5.7.0",
    "azure-storage": "^2.10.2"
```